### PR TITLE
[EBPF] Add Ubuntu 24.04 to KMT

### PR DESCRIPTION
**Pull request by gjulianm**
### What does this PR do?

Makes Ubuntu 24.04 available in the `platforms.json` file for KMT, so that teams can use it for CI.

It's not enabled by default as tests are failing for both system-probe and security-agent.

### CODE REVIEW

File: test/new-e2e/system-probe/config/platforms.json [hunk 1]
```diff
@@ -193,6 +193,17 @@
         "mantic"
       ]
     },
+    "ubuntu_24.04": {
+      "os_name": "Ubuntu",
+      "os_id": "ubuntu",
+      "kernel": "6.5.0-9-generic",
```
Comment (paulcacheux) [inline test/new-e2e/system-probe/config/platforms.json:199]: My understanding was that ubuntu 24.04 includes kernel 6.8, is this expected ?

Comment (gjulianm) [reply test/new-e2e/system-probe/config/platforms.json:199]: Yes, I just updated the paths manually so I could test the images, the kernel information is not updated yet. I'll do that once I ensure the image works correctly.

**Review by usamasaqib — APPROVED** [review]

**Pull request merged**